toffy/mph_comp.py

```diff
@@ -0,0 +1,128 @@
+import os
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+
+from mibi_bin_tools import bin_files
+from ark.utils import io_utils
+
+
+def compute_mph_metrics(bin_file_path, target, mass_start, mass_stop, save_csv=True):
```

Comment:
Remember, we want this to be compatible with the watcher. So there should be an FOV-level function that is self-contained and does everything related to generating the csv file.

See the qc metrics notebook for example: https://github.com/angelolab/toffy/blob/main/templates/3b_generate_qc_metrics.ipynb